README.md

```diff
@@ -11,51 +11,34 @@ Documentation is generated hourly from a scheduled task attached to the
 
 ### Rapid bootstrapping
 
-```shell
-# Global dependencies that are needed (optional if you have them already)
-# Installing python3 with Homebrew is the easiest way of getting what you need
-brew install python3
-pip3 install virtualenv
-
-# Clone repository
-git clone https://github.com/mapzen/mapzen-docs-generator.git
-cd mapzen-docs-generator
-
-# Install local dependencies
-make env
+On a Mac, assuming you have [Homebrew](http://brew.sh) and
+[Python 3](https://docs.python.org/3/using/mac.html) installed, and a local
+checkout of this repository:
 
-# Get all the sources
-make get
+```shell
+# Prepare virtualenv and install local dependencies
+virtualenv -p python3 venv
+source venv/bin/activate
+pip install -Ur requirements.txt
+npm install
```

Comment:
I think we can get rid of the `npm install` here and `package.json` -- it doesn't do anything anymore